tests/hardware_tests/test_mobile_gaze.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from spot_rl.envs.skill_manager import SpotSkillManager
+
+if __name__ == "__main__":
+    from spot_rl.utils.utils import map_user_input_to_boolean
+
+    # Pick Targets from Aria with +- 10 degree angle change
+    pick_targets = {
+        "kitchen": (
+            3.8482142244527835,
+            -3.4519528625906206,
+            np.deg2rad(-89.14307672622927),
+        ),
+        "kitchen_-10": (
+            3.8482142244527835,
+            -3.4519528625906206,
+            np.deg2rad(-89.14307672622927 - 10.0),
+        ),
+        "kitchen_+10": (
+            3.8482142244527835,
+            -3.4519528625906206,
+            np.deg2rad(-89.14307672622927 + 10.0),
+        ),
+        "table": (4.979398852803741, 3.535594946585519, -0.008869974097951427),
+        "table_+10": (
+            4.979398852803741,
+            3.535594946585519,
+            np.deg2rad(np.rad2deg(-0.008869974097951427) + 15.0),
+        ),
+        "table_-10": (
+            4.979398852803741,
+            3.535594946585519,
+            np.deg2rad(np.rad2deg(-0.008869974097951427) - 15.0),
+        ),
+    }
```

Comment:
can these be read from a file or config instead?

Reply:
@zephirefaith  addressed, will be reflected in next commit, moved these nav targets in waypoints.yaml